embassy/src/executor/raw/mod.rs

```diff
@@ -10,7 +10,7 @@
 mod run_queue;
 #[cfg(feature = "time")]
 mod timer_queue;
-mod util;
+pub(crate) mod util;
```

Comment:
I think this pub is not used?

Reply:
Ah, yes. I tried to use the UninitCell, but discovered it did not implement Sync + Send.